app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -600,11 +631,11 @@ private void goToRateApp() {
     }
   }
 
-  private void updateTitle(String zimFileTitle) {
+  private void updateTitle(String zimFileTitle, ActionBar actionBar) {
```

Comment:
I can't seem to figure out what this change has to do with dagger. Please only commit code that has to do with the PR at hand.